.github/workflows/ci.yml

```diff
@@ -4,6 +4,7 @@ on:
     branches:
       - main
   pull_request:
+  merge_group:
```

Comment:
Should we change this to:

```
on:
  merge_group:
    types: [checks_requested]
```

The Github docs suggests doing that to protect against future other types of events?